exo-go/src/service_restarter/index.go

```diff
@@ -0,0 +1,79 @@
+package serviceRestarter
+
+import (
+	"fmt"
+
+	"github.com/Originate/exosphere/exo-go/src/docker_compose"
+	"github.com/fsnotify/fsnotify"
+)
+
+// ServiceRestarter watches the given local service for changes and restarts it
+type ServiceRestarter struct {
+	ServiceName      string
+	ServiceDir       string
+	DockerComposeDir string
+	Env              []string
+	Log              func(string)
+	watcher          *fsnotify.Watcher
+}
+
+// Watch watches the service directory for changes
+func (s *ServiceRestarter) Watch(watcherErr chan<- error) error {
```

Comment:
Also since this is passed an error channel I don't think we need to return an error and can instead pass any error into the error channel.